ansible/roles/fips/tasks/redhat-7.yaml

```diff
@@ -0,0 +1,71 @@
+- name: "redhat 7 fips: ensure fips_enabled is 1"
+  lineinfile:
+    dest: /proc/sys/crypto/fips_enabled
+    line: "1"
+    state: present
+  become: yes
+  check_mode: yes
+  register: sysfips
+
+- name: "redhat 7 fips: disable prelink"
+  ansible.builtin.replace:
+    path: /etc/sysconfig/prelink
+    regexp: '^PRELINKING=yes'
+    replace: '^PRELINKING=no'
+  when:
+    - sysfips is changed
+
+- name: "redhat 7 fips: install dracut fips"
+  yum:
+    name:
+      - dracut-fips
```

Comment:
This package is not in the bundle, maybe we should only do this for >RHEL 8?

Reply:
we're doing it for rhel 7 thats it. i still haven't worked on testing this bit

Reply:
I meant not have this codepath at all for RHEL 7. Only do https://github.com/mesosphere/konvoy-image-builder/pull/508/files#diff-5f48c283af385dc86e531a1d0821000e9fee1c77cc9755ee4cc4c3f2c5b0eb76R1 for RHEL 8.

Reply:
removed